src/Api/Models/Response/ConfigResponseModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Bit.Core.Models.Api;
+using Bit.Core.Utilities;
+
+namespace Bit.Api.Models.Response
+{
+    public class ConfigResponseModel : ResponseModel
+    {
+        public ConfigResponseModel(string obj = "config") : base(obj)
+        {
+            Version = AssemblyHelpers.GetVersion();
+            Environment = new EnvironmentConfigResponseModel();
```

Comment:
Thoughts on having the constructor of `ConfigResponseModel` accept the global settings and assign the environment variables itself? It's a pattern we use in many other places.

Reply:
Yes, great idea. Added an overloaded constructor.